BridgeSDK/BridgeSDK.m

```diff
@@ -61,6 +65,20 @@ + (void)setupWithStudy:(NSString *)study useCache:(BOOL)useCache environment:(SB
     gSBBAppStudy = study;
     gSBBDefaultEnvironment = environment;
     gSBBUseCache = useCache;
+    gSBBCacheDaysAhead = SBBDefaultCacheDaysAhead;
+    gSBBCacheDaysBehind = SBBDefaultCacheDaysBehind;
+    
+    // make sure the Bridge network manager is set up as the delegate for the background session
+    [SBBComponent(SBBBridgeNetworkManager) restoreBackgroundSession:kBackgroundSessionIdentifier completionHandler:nil];
+}
+
++ (void)setupWithStudy:(NSString *)study cacheDaysAhead:(NSInteger)cacheDaysAhead cacheDaysBehind:(NSInteger)cacheDaysBehind environment:(SBBEnvironment)environment
+{
+    gSBBAppStudy = study;
+    gSBBDefaultEnvironment = environment;
+    gSBBUseCache = cacheDaysAhead > 0 || cacheDaysBehind > 0;
+    gSBBCacheDaysAhead = MIN(SBBMaxSupportedCacheDays, cacheDaysAhead);
```

Comment:
SBBMaxSupportedCacheDays is 30, but in SBBActivityManager, kMaxAdvance will cap this at 4 days. Is the 4 day restriction on what the server returns still valid? Depending on the answer, one value or the other should be used in both places.